### ISSUE TITLE
VID ADVZ: allow multiple evals per poly to reduce commitment communication

### ISSUE THREAD

**Issue by ggutoski** [EspressoSystems/jellyfish#356]
The [ADVZ scheme](https://eprint.iacr.org/2021/1500) specifies only 1 evaluation per polynomial. Thus, communication to storage nodes includes `k` polynomial evaluations + `k` polynomial commitments. As suggested by @chancharles92 we could reduce the number of polynomial commitments by a factor of `t` by allowing `t` evaluations per polynomial.

This will complicate the generation and verification of KZG proofs but it should still be practical. Currently we need only a batch proof + verification for evaluations of `k` polynomials at a single point `j`. This feature will require a batch proof for evaluations of `k` polynomials at multiple points `j_1,...,j_t`.

**Comment by chancharles92** [comment]
Let each node receive t evaluations per polynomial doesn't necessarily mean we need to change anything about the KZG generation/verification. For now, we can simply let each storage node receive t KZG individual evaluation proofs, one per evaluation point.

**Comment by ggutoski** [comment]
True. On the upside, we easily save space for polynomial commitments in the size-critical `Common` struct that is broadcast to all storage nodes. 🙂
